### ISSUE TITLE
Namespaced decorators "/*" argument are understood like multiline comment start

### ISSUE THREAD

**Issue by aguzubiaga** [TypeStrong/atom-typescript#539]
When you decorate with a namespaced decorator something and you set a parameter that contains "/*" atom starts multiline syntax highlighting:

![screenshot from 2015-08-22 14-43-02](https://cloud.githubusercontent.com/assets/1724813/9425152/1d1e0d6a-48dc-11e5-97b2-ba8c96cd2b00.png)

But that doesn't happend with unnamespaced decorators:
![screenshot from 2015-08-22 14-45-46](https://cloud.githubusercontent.com/assets/1724813/9425160/7ebd7c22-48dc-11e5-9008-ed0634d9eae0.png)

**Comment by Cu3PO42** [comment]
I'll look into this later.

**Comment by Cu3PO42** [comment]
While the problem itself is fixed, I am stuck on what I am inclined to say is an Atom bug:

![screen shot 2015-08-26 at 11 10 50](https://cloud.githubusercontent.com/assets/5256300/9489765/bd241a16-4be3-11e5-844a-67a8fbaf9d42.png)

As you can see, starting with the opening bracket the highlighting is shifted to the right by one character.

**Comment by nycdotnet** [comment]
any chance there is a BOM in the file?

**Comment by Cu3PO42** [comment]
There is not. However, this issue also only ever affects a single line at a time. It's all fine on the next line again. I intend to test on a few different Atom versions, still.

**Comment by aguzubiaga** [comment]
In my case it affect every line after the argument, however the typescript compiler and the linter works fine in those lines.
I am using atom 1.0.4.

**Comment by Cu3PO42** [comment]
Yes, the original problem was that the grammar doesn't accept dots in decorator names, however after fixing this every decorator appears to be broken (not just the namespaces ones), hence why I did not merge it yet.
